Modules/ThirdParty/NIFTI/UpdateFromUpstream.sh

```diff
@@ -8,13 +8,13 @@ readonly name="nifti"
 readonly ownership="NIFTI Upstream <nifti@github.com>"
 readonly subtree="Modules/ThirdParty/NIFTI/src/nifti"
 readonly repo="https://github.com/NIFTI-Imaging/nifti_clib.git"
-readonly tag="master"
```

Comment:
If this was close to 3.0, was it necessary to remove the files and create a new root commit?

Reply:
Because I had this error:
```
error: Entry 'Modules/ThirdParty/NIFTI/src/nifti/CMakeLists.txt' overlaps with 'Modules/ThirdParty/NIFTI/src/nifti/CMakeLists.txt'.  Cannot bind.
```

And the solution was to cleanup everything in #2626